whois-commons/src/main/resources/internals_schema.sql

```diff
@@ -103,6 +103,16 @@ CREATE TABLE `abuse_ticket` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
+
+DROP TABLE IF EXISTS `default_maintainer_sync_history`;
+CREATE TABLE `default_maintainer_sync_history` (
+    `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `org` varchar(256) NOT NULL,
+    `mntner` varchar(256) NOT NULL,
+    `timestamp` timestamp NOT NULL DEFAULT current_timestamp() ON UPDATE current_timestamp(),
+    `email` varchar(256) NOT NULL,`is_synchronize` tinyint(1) DEFAULT 0,PRIMARY KEY (`id`)
```

Comment:
Where is 'is_synchronise' column?
Can you rename is_synchronise to 'synchronised' or 'is_synchronised' to be grammatically correct.

Reply:
Where is 'is_synchronise' column? - forgot to put enter  after email column.
Will rename the column to synchronised and will make changes in whois-internal as well

Reply:
Great, thanks!